### ISSUE TITLE
Check both neovim versions (nightly and stable) in the CI

### ISSUE THREAD

**Issue by kylo252** [LunarVim/LunarVim#2348]
### Discussed in https://github.com/LunarVim/LunarVim/discussions/2250

<div type='discussions-op-text'>

<sup>Originally posted by **starptr** February  4, 2022</sup>
Hi, hope you're having a good day!
</br>

Currently, it looks like even the stable releases are tested on a "rolling basis" due to the practice of most people using neovim nightlies. I was wondering if it was possible for the releaser to also tell us which neovim versions (stable, a specific commit build, or otherwise) have been tested every time there is a new lunarvim release? That way, I don't try to use Lunarvim against an untested neovim build 😅
</br>

Of course, ideally I will submit a bug report for such issues; however, in many cases I need to get work done before I mess around with Lunarvim itself. Although I love the direction Lunarvim is taking, bugs arising from using untested neovim builds make it difficult for me to continue using it when I just need to get stuff done 😢.
</br>

TL;DR: whenever there is a Lunarvim stable release, please tell us which neovim builds have been tested against the particular release!</div>
